diff --git a/runtime/graphdriver/btrfs/btrfs.go b/runtime/graphdriver/btrfs/btrfs.go
index 592e058..03faf85 100644
--- a/runtime/graphdriver/btrfs/btrfs.go
+++ b/runtime/graphdriver/btrfs/btrfs.go
@@ -30,7 +30,7 @@ func Init(home string) (graphdriver.Driver, error) {
 		return nil, err
 	}

-	if buf.Type != 0x9123683E {
+	if buf.Type != -1859950530 {
 		return nil, fmt.Errorf("%s is not a btrfs filesystem", rootdir)
 	}

--
1.9.0
